src/components/10-atoms/checkbox/demo.js

```diff
@@ -12,25 +12,18 @@ storiesOf('Components|Atoms/Checkbox/Demos', module)
   })
   .add('Feature - Use your own label as a child of the component', () => {
     const template = html`
-      <div>
-        <axa-checkbox
-          class="hover"
-          name="checkbox"
-          checked="false"
-          required="true"
-          ><p>click <a href="https://www.google.ch" target="_blank">here</a></p>
-        </axa-checkbox>
-        <axa-checkbox
-          class="hover"
-          name="checkbox"
-          checked="false"
-          required="true"
-          ><p>
-            or click <a href="https://www.google.ch" target="_blank">here</a> if
-            you prefere
-          </p>
-        </axa-checkbox>
-      </div>
+      <axa-checkbox
+        class="hover"
+        name="checkbox"
+        checked="false"
+        required="true"
+        ><p>
+          I agree to
```

Comment:
"I agree to the", right?

Reply:
axa.ch shows it without "the".

![Screenshot 2020-01-31 at 15 21 43](https://user-images.githubusercontent.com/38284563/73546381-6d8ef280-443d-11ea-9530-2ff64a82f4c8.png)